recipe/meta.yaml

```diff
@@ -9,37 +9,37 @@ package:
 source:
   fn: {{ version }}.tar.gz
   url: https://github.com/santiagoropb/pynumero_libraries/archive/v{{ version }}.tar.gz
-  sha256: cf21b5045aa18c636b4e2e13fe4f42119b0a4b0ccef96e43d4e5876f675d78a7
+  sha256: d08985d7832e30db6d45998399178cd3d4b96db6931ec400107066a354a7f066
     
 build:
   number: 0
-  skip: true  # [win]
-      
+  skip: true  # [win and vc<14]
+  
 requirements:
   build:
     - cmake
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - ampl-mp
-    - python
-    - numpy
-    - pymumps
-    - matplotlib
-    - scipy
+    - ampl-mp  # [not win]
```

Comment:
These dependencies are not available for Windows I guess?

Reply:
It seems that cmake then vendors them, at least I see some things in `third-party` that get cloned/compiled. Is that intentional?

Reply:
They are not. However, the CMake project on windows builds those dependencies instead of pulling them from conda-forge

Reply:
It's usually a bad idea to vendor dependencies. It will usually cause trouble further down the road.